### PR TITLE
cut and clip leave ghosts; static view defaults to no-grid

### PR DESCRIPTION
**Pull request by pentacular**

### CODE REVIEW

File: es6/jsxcad-ui-threejs.js [hunk 1]
```diff
@@ -619,7 +619,13 @@ const merge = async (properties, parameters) => {
 };
 
 const setMaterial = async (definitions, tags, parameters) => {
-  const threejsMaterial = toThreejsMaterialFromTags(tags, definitions);
+  let threejsMaterial;
+  if (tags.includes('type:ghost')) {
+    // FIX: This is ugly.
```
Comment (BarbourSmith) [inline es6/jsxcad-ui-threejs.js:624]: How so?

File: es6/jsxcad-ui-threejs.js [hunk 2]
```diff
@@ -1681,7 +1689,7 @@ const staticDisplay = async (
     canvas,
     geometry,
     withAxes = false,
-    withGrid = true,
+    withGrid = false,
```
Comment (BarbourSmith) [inline es6/jsxcad-ui-threejs.js:1692]: Is the grid gone by default now?

**Review by BarbourSmith — COMMENTED** [review]
Aren't you supposed to be offline this weekend? 😀 

I like how the ghost looks and I'm excited to see how it feels in practice.

There are a couple of these which are a little concerning

image

Well this is cool:

image

But this doesn't look right, does it?

image

Or this:

image

File: es6/jsxcad-ui-threejs.js (hunk 1 above)
Comment (pentacular) [reply es6/jsxcad-ui-threejs.js:624]: Well, it's just hacking in an arbitrary material if it's a ghost geometry. Actually, a material should probably be set on creation (and transparency set likewise)

File: es6/jsxcad-ui-threejs.js (hunk 2 above)
Comment (pentacular) [reply es6/jsxcad-ui-threejs.js:1692]: Yeah, I wanted to see how it looks, and if it helps reduce noise in the diffs.

But an explicitly provided value should be honored.

Comment (BarbourSmith) [reply es6/jsxcad-ui-threejs.js:1686]: I think that makes sense. On the static views I rarely use the grid, and if I wanted to use it I would probably want the top down view

**Review by BarbourSmith — APPROVED** [review]
Is this OK?

image

Same here, does these look right?

image

image

If those look good to you then it all looks good to me!

**Pull request merged**